cli/app/app.go

```diff
@@ -925,23 +925,27 @@ type convertOptions struct {
 }
 
 // Convert komposeObject to K8S controllers
-func komposeConvert(komposeObject KomposeObject, opt convertOptions) (map[string][]byte, map[string][]byte, map[string][]byte, map[string][]byte, map[string][]byte, []string) {
-	mServices := make(map[string][]byte)
-	mReplicationControllers := make(map[string][]byte)
-	mDeployments := make(map[string][]byte)
-	mDaemonSets := make(map[string][]byte)
-	// OpenShift DeploymentConfigs
-	mDeploymentConfigs := make(map[string][]byte)
-
+func komposeConvert(komposeObject KomposeObject, opt convertOptions) []runtime.Object {
 	var svcnames []string
 
+	// this will hold all the converted data
+	var objects []runtime.Object
 	for name, service := range komposeObject.ServiceConfigs {
 		svcnames = append(svcnames, name)
-		rc := initRC(name, service, opt.replicas)
 		sc := initSC(name, service)
-		dc := initDC(name, service, opt.replicas)
-		ds := initDS(name, service)
-		osDC := initDeploymentConfig(name, service, opt.replicas) // OpenShift DeploymentConfigs
+
+		var controller runtime.Object
+		// only one controller will be created per service so using switch
```

Comment:
I am confused here, why would we have multiple controllers when `--stdout` or `--out` is not given?